code/game/objects/items.dm

```diff
@@ -105,6 +105,8 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 	var/datum/rpg_loot/rpg_loot = null
 
+	var/held = null

```

Comment:
but logically and item can be spawned into a mob's hand, and that item wouldn't trigger any of the checks to set held to TRUE. Making it initialize to FALSE would be incorrect in some situations.